### ISSUE TITLE
Discretization of continuous features for explanations

### ISSUE THREAD

**Issue by NMRobert** [TeamHG-Memex/eli5#207]
Hi,

What are the challenges involved in implementing discretization for cts features such that we can estimate the probability of a classifier returning a particular class conditioned on the cts feature(s) taking on particular ranges of values? I believe that something similar to this exists for LIME. In the Titanic xgboost example, this would mean that instead of finding out that for example 'Age' is an influential variable with no further context, we might determine that Age in the first quartile of ages (ie: maybe (0,15)) has a large positive effect on being saved, and being in the last quartile has a large negative effect, etc.  Currently it seems (although I may be wrong) that the explanations mostly only apply to categorical features. Cheers!

**Comment by kmike** [comment]
Yeah, that would be a nice addition.
